### ISSUE TITLE
Document clearly how to get started with svg.js

### ISSUE THREAD

**Issue by David263** [svgdotjs/svg.js#954]
Currently, the only documentation for this library seems to be https://svgjs.com/docs/2.7/ . However, there is an unnecessary hurdle for a newcomer in its structure (and possibly content).

First, there is no help whatsoever. No community forum. No support tool. No "contact us" by email. Not even an old-fashioned mailing list. Nothing. If these things exist, they are well hidden.

Next, how about making and embedding on the homepage a codepen or jsfiddle startup example, displaying something simple like a circle? And another codepen displaying the full range of complex shapes, etc., achievable with svg.js?

When I try to write my own program, I get the error "SVG is undefined". I did not expect this after adding

```
<div id="drawing"></div>
<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/svg.js/3.0.11/svg.min.js"></script>
```

to my HTML and

```
	var draw = SVG('drawing').size(500, 50);
	var text = draw.text('I know that eggs do well to stay out of frying pans.');
	text.move(20,20).font({ fill: '#f06', family: 'Inconsolata' });
```
to my JavaScript, following the current scattered documentation as well as I could.

**Comment by Fuzzyma** [comment]
### TL;DR

see https://github.com/svgdotjs/svg.js/issues/948#issuecomment-450647545

Every time you open a new bug, we ask you first, if it is a question for stackoverflow. This seems like such a case. Everything else you wrote is a rant about us developers. We have a chat on gitter where we try to help everyone. We also answer questions on stackoverflow as soon as we can. We really try to update the documentation as soon as we can. 

Version 3 works a tiny bit different then 2. You create a new SVG doc by simply calling `SVG()` and appending it to a container. e.g. `SVG().addTo('body')` or in your case `addTo('#drawing')`.
After that most of the api is the same and the docs for 2.7 are often still valid.

**Comment by PiotrFr** [comment]
v. 2.7     var element = SVG.get('my_element')
v. 3.0     var element = ?

**Comment by Fuzzyma** [comment]
You would use `SVG(css Selector)` e.g. `SVG('#id')`. For multiple elements you would use `SVG.find(selector)`

**Comment by PiotrFr** [comment]
Thank you.
`var id = element.id()`
v. 2.7  `var element2 = SVG(id)`
v. 3.0  `var element2 = SVG('#' + id)`    or    `var element2 = SVG.find('#' + id)[0]`

**Comment by nikitakotenko88** [comment]
With v2.5 it's ok, but v3 say "stop.at is not a function" when I try define the gradient.
`
<svg id="svg1"> </svg>

var draw = SVG('#svg1').size(100,100);
var linear = draw.gradient('linear', function(stop) {
  stop.at(0, '#f06', 1);
  stop.at(1, '#0f9', 1);
});

draw.rect(100, 100).fill( linear );
`

**Comment by Fuzzyma** [comment]
Yes, another change here. Basically the constructor for the stop-element was renamed to `stop` for consistency:

```js
var linear = draw.gradient('linear', function(add) {
  add.stop(0, '#f06', 1);
  add.stop(1, '#0f9', 1);
});
```

**Comment by nikitakotenko88** [comment]
Thank You alot! Waiting for update of API docs!

**Comment by nathanielw** [comment]
@Fuzzyma I completely understand that there's usually a desire to get new features and fixes out quickly, but it seems counter-intuitive to release a new version if very few people are going to know how to use it or migrate their existing code. By no means do I intend for this to come across as a rant or attack. I'm hugely appreciate all the effort that's been put into the library and understand that a lot of work is probably going into updating the documentation.

To alleviate some of the frustration new users seem to be facing, would you consider adding a note in the readme and on the website stating that v2.7 is still the recommended version for newcomers while work to update the documentation is underway, and perhaps link to a ticket where progress on the docs can be viewed?

As a suggestion for making releases a smoother transition in the future, I'm a fan of the approach taken by a lot of big JS libraries where a release isn't marked as the latest (or considered stable) until the documentation is up-to-date, and generally pull-requests aren't accepted if they don't include relevant documentation updates.

Thanks again for all your work on this project :)

**Comment by Fuzzyma** [comment]
@nathanielw Yes, you are right. However, the intention wasnt to get features out as soon as possible. We released the new version because else it would have never happened. After 2 years of working and people waiting for it I dont have the time and strength to put another year into the documentation. I know that this is not ideal but thats how it is.
An update of the readme is a good step. Also making a note on the current docs. I thing I can do that soon. However, for complete docs you may have to wait for another while.

**Comment by nathanielw** [comment]
Thanks for the response and context @Fuzzyma, really appreciate it :)

**Comment by ostanioni** [comment]
Hello.
How to use SVG.js with Webpack?
1. `import * as _SVG from '@svgdotjs/svg.js'`
2. `const canvas = _SVG.SVG().addTo('#drawing').size(300, 300)
    var rect = canvas.rect(100, 100).attr({ fill: '#f06' })`
maybe there is a better way

**Comment by Fuzzyma** [comment]
Yes you can import everything like that. However, you can aso use named exports and only import what you need: `import {SVG} from '@svgdotjs/svg.js'`.
The imports you get are the same which were former available under the big global `SVG` object. The `SVG()` function itself is also imported now (as seen in your and my example).
If you want the big fat object back, you can instead import another file:

```js
import SVG from '@svgdotjs/svg.js/src/svg.js'
```

This will give you said object. However, named exports might give you a benefit with treeshaking.

**Comment by ostanioni** [comment]
Thank you for your prompt response
the problem was that it was necessary to use
`SVG().addTo('#drawing').size(300, 300)`
instead
`SVG('drawing').size(300, 300)`

**Comment by philmobley** [comment]
Current code on 3.0 getting started doesn't even work... only asking to update the documentation to reflect what is in this thread instead.

```js
var draw = SVG('drawing').size(300, 300)
var rect = draw.rect(100, 100).attr({ fill: '#f06' })
```

Even the playground at the bottom is actually using v2.5 code that works as written, so its significantly more confusing to the new user:

```html
<head>
  ...
  <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/svg.js/2.5.0/svg.js"></script>
  ...
</head>
```

**Comment by theBull** [comment]
> Thank you for your prompt response
> the problem was that it was necessary to use
> `SVG().addTo('#drawing').size(300, 300)`
> instead
> `SVG('drawing').size(300, 300)`

This worked for me as well.

**Comment by avenmore** [comment]
Wow, half a day wasted until I found this issue with information that helped. A lesser man would have given up. To get it working in my Vue + Typescript component:

```
  import { Svg } from '@svgdotjs/svg.js';

            const draw = new Svg().addTo('#drawing').size('300px', '300px');
            const rect = draw.rect(100, 100).attr({ fill: '#f06' });

```

**Comment by sytpb** [comment]
`<!DOCTYPE html>
<html>
<head>
  <title>SVG.js</title>

<script  src="https://cdnjs.cloudflare.com/ajax/libs/svg.js/3.0.13/svg.js">

    var draw = SVG().addTo('drawing').size(500, 50);
    var text = draw.text('I know that eggs do well to stay out of frying pans.');
    text.move(20,20).font({ fill: '#f06', family: 'Inconsolata' });
</script>
</head>
<body>

<div id="drawing"></div>
</body>

</html>`

web Got nothing  , why ?  where is wrong .

**Comment by sytpb** [comment]
`            <div className={classes.inlineRight}>
              <Typography style={{ textTransform: 'uppercase' }} color='secondary' gutterBottom>
                Other Amount
              </Typography>
              <Typography variant="h4" gutterBottom>
                Once a month
              </Typography>
              <ButtonBar />
            </div>
          </div>`

**Comment by sytpb** [comment]
`<!DOCTYPE html>
<html>
<head>
  <title>SVG.js</title>

<script src="https://cdnjs.cloudflare.com/ajax/libs/svg.js/3.0.13/svg.js"></script>


</head>
<body>

<div id="drawing"></div>
</body>



<script>
    //var draw = SVG().addTo('#drawing').size(300, 300)
    var draw = SVG('drawing').size(300, 300)
    var text = draw.text('I know that eggs do well to stay out of frying pans.');
    text.move(20,20).font({ fill: '#f06', family: 'Inconsolata' });
</script>
</html>`

**Comment by sytpb** [comment]
![图片](https://user-images.githubusercontent.com/12178686/67662319-24428900-f99e-11e9-929c-bd2d52ca9774.png)

it did nｏt work , where is wrong ?

**Comment by Fuzzyma** [comment]
I finally updated the docs. Thanks for your patience!
